spec/features/idv/previous_address_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature 'IdV with previous address filled in' do
+  include IdvHelper
+
+  let(:bad_zipcode) { '00000' }
+
+  it 'fails when either address has bad value, prefers current address in profile' do
+    user = sign_in_and_2fa_user
+    visit verify_session_path
+    fill_out_idv_form_ok
+    fill_out_idv_previous_address_fail
+    click_idv_continue
+
+    expect(current_path).to eq verify_session_path
+    expect(page).to have_selector("input[value='#{bad_zipcode}']")
+
+    fill_out_idv_previous_address_ok
+    fill_out_idv_form_fail
+    click_idv_continue
+
+    expect(current_path).to eq verify_session_path
+    expect(page).to have_selector("input[value='#{bad_zipcode}']")
```

Comment:
oh yeah! I do think that previous one was easier to understand. I love fast tests, but I love readable tests even more.

Maybe a compromise is to keep the big test but group sets of steps into methods so it can be skimmed more easily?